durango/src/error.rs

```diff
@@ -0,0 +1,102 @@
+//! The Durango error
```

Comment:
This file should not be included. It is possibly a rebase mistake either in your branch or any previous merge requests.